src/schema/v2/auction_result.ts

```diff
@@ -141,6 +141,39 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    performance: {
+      type: new GraphQLObjectType({
+        name: "AuctionLotPerformance",
+        fields: {
+          mid: {
+            type: GraphQLString,
+            description: "Percentage performance over mid-estimate",
+            resolve: ({
+              hammer_price_cents,
+              high_estimate_cents,
+              low_estimate_cents,
+            }) => {
+              if (
+                hammer_price_cents &&
+                high_estimate_cents &&
+                low_estimate_cents
+              ) {
+                return (
+                  Math.round(
+                    ((hammer_price_cents -
+                      (low_estimate_cents + high_estimate_cents) / 2) /
+                      hammer_price_cents) *
```

Comment:
I'd expect the mid-estimate to be the denominator here, since we're calculating the percentage _over mid estimate_. 

If the estimate was between 10 and 30, the mid-estimate was 20. If the realized price was 100, I'd expect that to be significantly over mid-estimate (500%). With the current calculation, with 100 in the denominator, the percentage would instead come out quite muted.

Reply:
thanks Steve